analyze/action.yml

```
@@ -79,6 +79,8 @@ inputs:
 outputs:
   db-locations:
     description: A map from language to absolute path for each database created by CodeQL.
+  output:
```

Maybe `sarif-output` would be more explicit here.
```suggestion
  sarif-output:
```